test-tool/test_async_write.c

```diff
@@ -58,7 +67,7 @@ void
 test_async_write(void)
 {
 	int i, ret;
-	struct tests_async_write_state state = { 0, 0, 0 };
+	struct tests_async_write_state state = { 0, 0, 0, 0, 0 };
```

Comment:
How about using "{ }" to zero-initialize this data structure?

Reply:
Sure, works for me, although I'd had complaints against implicit zeroing in the past (see 71879c913a91db43d8317089979ac2ca2d84dc60)

Reply:
It seems that the clang bug referred to in the commit (https://bugs.llvm.org/show_bug.cgi?id=21689) has since been fixed.